types/event_test.go

```diff
@@ -131,13 +131,13 @@ func TestEventIsResolution(t *testing.T) {
 				CheckHistory{Status: 1},
 			},
 			status:   0,
-			expected: true,
+			expected: false,
 		},
 		{
 			name: "check has transitioned but still an incident",
 			history: []CheckHistory{
-				CheckHistory{Status: 0},
 				CheckHistory{Status: 2},
+				CheckHistory{Status: 0},
```

Comment:
Based on the comment above for `IsResolution()`, I don't see why, if the history includes the current check, `Check.Status` would be different than `Check.History[len(Check.History) - 1].Status`

Reply:
Ha my bad!